platform-api/che-core-api-builder/src/main/java/org/eclipse/che/api/builder/BuilderSelectionStrategy.java

```diff
@@ -22,4 +23,8 @@
  */
 public interface BuilderSelectionStrategy {
     RemoteBuilder select(List<RemoteBuilder> slaveBuilders);
+
+    default RemoteBuilder select(List<RemoteBuilder> slaveBuilders, BaseBuilderRequest request){
+        return select(slaveBuilders);
+    }
```

Comment:
Thanks @skabashnyuk 
I'll check how to add this and create another pull next week